app/assets/stylesheets/pages.css.scss

```diff
@@ -3,50 +3,78 @@
 $include-html-global-classes: false;
 @import "foundation/components/global";
 
+::selection {
+  background: $hackclub-red;
+  color: $hackclub-white;
+  opacity: 1;
+  text-shadow: none;
+}
+
+::-moz-selection {
```

Comment:
Cleared. Ignored vendor fix because it's required for ::selection and many newer web technology.